src/app/components/pages/LogIn.tsx

```diff
@@ -182,21 +184,19 @@ export const EmailPasswordInputs =({setEmail, setPassword, validEmail, validPass
         </FormGroup>
 
         <FormGroup className="mb-0">
-            {displayLabels && (<><Label htmlFor="password-input">Password</Label>
+            {displayLabels && (<><Label htmlFor="current-password">Password</Label>
             <span id={`password-help-tooltip`} className="icon-help ml-1" />
             <UncontrolledTooltip target={`password-help-tooltip`} placement="bottom">
             {PASSWORD_REQUIREMENTS}
             </UncontrolledTooltip></>)}
-            <Input
-                id="password-input" autoComplete="current-password" type="password" name="password" placeholder="Password"
+            <Password
+                passwordFieldType="Current"
+                isPasswordVisible={isPasswordVisible}
+                setIsPasswordVisible={setIsPasswordVisible} 
                 onChange={(event: React.ChangeEvent<HTMLInputElement>) => setPassword(event.target.value)}
                 invalid={!!errorMessage || (!validPassword && (logInAttempted))}
-                aria-describedby="passwordValidationMessage"
-                required
+                showToggleIcon={true}
             />
-            <FormFeedback id="passwordValidationMessage">
```

Comment:
This `FormFeedback` is gone. Was this intentional?

Reply:
Yep that's intentional to fix a current bug I noticed. We changed the password requirements, but we've grandfathered them in, so currently if a user tries to login with an old password ("test1234" for example) it will allow them to login but it will also show the password feedback briefly, which looks awful.  So I removed the PW feedback - the PW requirements are in a tooltip on that page anyway, and they get an error back from the API if the PW is wrong.